_activities/sinkfloat.md

```diff
@@ -1,12 +1,21 @@
 ---
 layout: activity
-lang: en
 #
 # Content
 #
 title: "Sink or Float"
+title_fr: “”
 age: 8-12
 objective: "Discover how factors like mass, volume, and shape affect the ability of an object to float."
+objective_fr: “Découvrir comment les facteurs tels que la masse, le volume et la forme affectent la capacité d’un objet de flotter.”
+summary: “In this activity, students will be introduced to the scientific method which is a technique used in experimentation based on empirical evidence. The students will use this method to test their hypotheses and to determine that density affects an object’s ability to float. ”
+summary_fr: “Dans cette activité, les élèves seront introduits à la méthode scientifique, une technique utilisée dans l'expérimentation basée sur des preuves empiriques. Les élèves  utiliseront cette méthode pour vérifier leurs hypothèses afin de conclure que la densité affecte la capacité d’un object de flotter.”
+image:
+   thumb:
+   title:
+downloads:
+   name: “”
+   filename: “”
```

Comment:
Add downloadables - there's the facilitator guide (filename `Sink or Float - facilitator guide_EN.pdf`) and english worksheet (filename `Sink or Float - student worksheet_EN.pdf`) in the downloads folder. The format is also not quite right; for example (also can copy/paste from "collisions.md"):
`
downloads:
  - 
    name: "Facilitator guide"
    filename: "Sink or Float - facilitator guide_EN.pdf"
`